### PR TITLE
[bonopark] Added Madrid's BiciMAD network

### PR DESCRIPTION
**Pull request by bcaller**
The bike network for Madrid, Spain.

### CODE REVIEW

File: pybikes/bonopark.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+
+    def __init__(self, tag, meta, url):
+        super(Bonopark, self).__init__(tag, meta)
+        self.feed_url = url
+
+    def update(self, scraper=None):
+        if scraper is None:
+            scraper = utils.PyBikesScraper()
+        scraper.headers['Content-Type'] = 'application/json'
```
Comment (eskerda) [inline pybikes/bonopark.py]: Can we set this user agent  `Apache-HttpClient/UNAVAILABLE (java 1.4)`? Seems to be the one the app is using.

Comment (bcaller) [inline pybikes/bonopark.py]: Yes that would be the correct user agent, though for now it works without.

**Comment by bcaller** [comment]
I have added the user agent.

**Comment by eskerda** [comment]
Looks good, thanks

**Pull request merged**